test/system/system.test.ts

```diff
@@ -595,18 +590,6 @@ test("inspect centos", (t) => {
         },
         "deps",
       );
-
-      t.match(
-        manifest,
-        [
-          {
-            name: "redhat-release",
-            path: "/etc",
-            contents: "Q2VudE9TIExpbnV4IHJlbGVhc2UgNy40LjE3MDggKENvcmUpIAo=",
-          },
-        ],
-        "manifest files",
-      );
```

Comment:
Is this removed unintentionally?

Reply:
yes, I'm not sure if the multiple response object gets processed in the CLI to even preserve this list, and even if it does I believe having them there is incorrect